server/src/test/java/org/apache/druid/server/metrics/TestDynamicConfigurationManager.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server.metrics;
+
+import com.google.common.collect.ImmutableSet;
+import org.apache.druid.client.DynamicConfigurationManager;
+import org.apache.druid.server.coordinator.CoordinatorDynamicConfig;
+
+import java.util.Set;
+
+public class TestDynamicConfigurationManager extends DynamicConfigurationManager
```

Comment:
I think the concrete implementation `DynamicConfigurationManager` is simple enough to not require a dedicated test implementation.

Reply:
Removed.